Sources/VariadicsGenerator/VariadicsGenerator.swift

```diff
@@ -118,6 +118,17 @@ struct VariadicsGenerator: ParsableCommand {
     precondition(maxArity < Counter.bitWidth)
 
     output("""
+      //===----------------------------------------------------------------------===//
+      //
+      // This source file is part of the Swift.org open source project
+      //
+      // Copyright (c) 2021-2022 Apple Inc. and the Swift project authors
+      // Licensed under Apache License v2.0 with Runtime Library Exception
+      //
+      // See https://swift.org/LICENSE.txt for license information
+      //
+      //===----------------------------------------------------------------------===//
+
```

Comment:
@rxwei Does it make sense to include the copyright notice here? As otherwise it gets removed when re-running the generator.

Reply:
Yes it does. I'm already adding it here: https://github.com/apple/swift-experimental-string-processing/pull/114, but the PR is blocked until there's a new toolchain.

Reply:
It might be better to change this so that I won't have to resolve conflicts for my chain of 3 PRs 😂

Reply:
Yeah I was just thinking that, though I think there will still be a conflict as I'm still changing Concatenation.swift, but at least that can be resolved by re-running the generator